src/widget/systemtrayicon.cpp

```diff
@@ -233,13 +244,17 @@ void SystemTrayIcon::setContextMenu(QMenu* menu)
                 };
                 QImage* image = new QImage(a->icon().pixmap(64, 64).toImage());
                 if (image->format() != QImage::Format_RGBA8888_Premultiplied)
-                *image = image->convertToFormat(QImage::Format_RGBA8888_Premultiplied);
+                    *image = image->convertToFormat(QImage::Format_RGBA8888_Premultiplied);
                 GdkPixbuf* pixbuf = gdk_pixbuf_new_from_data(image->bits(), GDK_COLORSPACE_RGB, image->hasAlphaChannel(),
```

Comment:
But the ```image``` won't get deleted until after ```pixbuf```'s reference count has dropped to zero, as per [docs](https://developer.gnome.org/gdk-pixbuf/stable/gdk-pixbuf-Image-Data-in-Memory.html#gdk-pixbuf-new-from-data). Note that the deletion only happens in the callback passed to ```gdk_pixbuf_new_from_data()```.